### PR TITLE
Add more services

### PR DESCRIPTION
**Pull request by toabctl**
Allow to deploy OpenStack Sahara.

### CODE REVIEW

**Comment by toabctl** [comment]
UNTESTED!

**Comment by Itxaka** [comment]
Looks good but I dont know the quickstart scripts too well :)

File: scripts/openstack-quickstart-demosetup [hunk 1]
```diff
@@ -696,7 +696,8 @@ if [ "x$with_manila" = "xyes" ]; then
     start_and_enable_service openstack-manila-scheduler
     start_and_enable_service openstack-manila-share
     start_and_enable_service openstack-manila-data
-
+    # wait a bit until manila-api is ready
+    sleep 5
```
Comment (dirkmueller) [inline scripts/openstack-quickstart-demosetup]: not particularly happy about sleeps.. can we add a small loop around manila type-list ?

Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: The real fix would be fixing manila to use the SD_NOTIFY interface from systemd. But I can add the loop

Comment (toabctl) [inline scripts/openstack-quickstart-demosetup]: done

**Pull request merged**